### ISSUE TITLE
removeAll typing is wrong

### ISSUE THREAD

**Issue by TheDutchCoder** [microcipcip/cookie-universal#137]
The docs state that you can supply options, but the Typing says you can't: `removeAll: () => void`

**Comment by microcipcip** [comment]
You're right, I'll fix it. The docs are also wrong because only `path` prop makes sense there.
